### PR TITLE
Add recipe for inference-server

### PR DESCRIPTION
**Pull request by faph**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by faph** [comment]
Hello @conda-forge/help-python this is now ready for review.

**Comment by faph** [comment]
@conda-forge-admin, please ping team

File: recipes/inference-server/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/inference_server-{{ version }}.tar.gz
+  sha256: 08bc663404dbddb160b7cd8cf9f507b0baed8cd3271d66baf11c1f1f939a58ac
+
+build:
+  skip: true  # [py<38]
```
Comment (ocefpaf) [inline recipes/inference-server/meta.yaml:13]: Looks like this package can be noarch, right?

https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

**Comment by faph** [comment]
> Hi! This is the friendly automated conda-forge-linting service.
> 
> I wanted to let you know that I linted all conda-recipes in your PR (`recipes/inference-server/meta.yaml`) and found some lint.
> 
> Here's what I've got...
> 
> For **recipes/inference-server/meta.yaml**:
> 
>     * `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.

I thought I removed the selectors? Could you please advise @ocefpaf

File: recipes/inference-server/meta.yaml [hunk 2]
```diff
@@ -11,19 +11,18 @@ source:
...
     - setuptools >=45
     - wheel
     - setuptools-scm >=6.2
     - pip
   run:
-    - python
+    - python >=3.9
     - botocore
     - codetiming >=1.4,<2.dev0
     - importlib-metadata <4  # [py<38]
```
Comment (ocefpaf) [inline recipes/inference-server/meta.yaml:28]: It is OK to install this in py>=38, the standard lib version will be used instead.

```suggestion
    - importlib-metadata <4 
```

Comment (faph) [reply recipes/inference-server/meta.yaml]: Ah yes, missed that one. All fixed.

**Comment by faph** [comment]
I think we’re all good now @ocefpaf ?

**Pull request merged**